functions/src/utils/venue.js

```diff
@@ -0,0 +1,12 @@
+const getVenueId = (name) => {
+  return name.replace(/\W/g, "").toLowerCase();
+};
+
+const checkIfValidVenueId = (venueId) => {
+  if (typeof venueId !== "string") return false;
+
+  return /[a-z0-9_]{1,250}/.test(venueId);
+};
```

Comment:
[TODO] Extract the `venueId` regex as a const
[TODO] Make a `yup` schema for `VenueId`: https://github.com/jquense/yup#stringmatchesregex-regex-message-string--function-schema

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1523/commits/b1a0c754fa2a8deae8481d5e1458786e05b42428 + https://github.com/sparkletown/sparkle/pull/1523/commits/660cbc74e22012af2a7d5c753ff092717664a131